rust/src/dcerpc/dcerpc.rs

```diff
@@ -1293,7 +1293,7 @@ pub unsafe extern "C" fn rs_dcerpc_probe_tcp(_f: *const core::Flow, direction: u
                                       len: u32, rdir: *mut u8) -> AppProto
 {
     SCLogDebug!("Probing packet for DCERPC");
-    if len == 0 {
+    if len == 0 || input.is_null() {
```

Comment:
Do we have a behavior change here we should be concerned about?

Currently if input is null we create an empty slice which I believe will drop down to a return value of `ALPROTO_FAILED`, but now we're returning unknown.

Reply:
Most protocols, including the template, return `ALPROTO_UNKNOWN` if `input.is_null()`

So, looks like a good generalization to me...